### ISSUE TITLE
Issue about backward

### ISSUE THREAD

**Issue by lzr-thierry** [vlomonaco/crlmaze#1]
Is it necessary to set requires_grad=True for ewc loss? It seems that the gradient of ewc is not considered in backward.

**Comment by vlomonaco** [comment]
I answered you via email!
